extension-style/api/extension-style.api

```diff
@@ -3411,16 +3411,16 @@ public final class com/mapbox/maps/extension/style/sources/TileSet : java/util/H
 	public fun containsValue (Lcom/mapbox/bindgen/Value;)Z
 	public final fun containsValue (Ljava/lang/Object;)Z
 	public final fun entrySet ()Ljava/util/Set;
-	public final fun get (Ljava/lang/Object;)Ljava/lang/Object;
+	public final fun get (Ljava/lang/Object;)Lcom/mapbox/bindgen/Value;
```

Comment:
interesting, is this related to version bump?

Reply:
yeah looks like it. this is edited by kotlin binary compatibility check and verify-api also caught it as a binary breaking change (check the internal ticket regarding breaking changes). @pengdev any comment on this.

Reply:
The TileSet extends a `HashMap<String, Value>`.

The difference seems to be that the previous API, the type is erased, while the new API keeps the type.
This might be a compiler behaviour change. And I think the `com/mapbox/bindgen/Value` should be the correct type that java users could safely reference. I think we could consider this as a bug fix and not an API breaking change, wdyt?

Reply:
sounds good to me 👍